### ISSUE TITLE
Error when running train.py

### ISSUE THREAD

**Issue by supriyalk** [SeanNaren/deepspeech.pytorch#104]
using the command python train.py --train_manifest data/an4_train_manifest.csv --val_manifest data/an4_val_manifest.csv

I get a seg fault error: 
```
Directory already exists.
DeepSpeech (
  (conv): Sequential (
    (0): Conv2d(1, 32, kernel_size=(41, 11), stride=(2, 2))
    (1): BatchNorm2d(32, eps=1e-05, momentum=0.1, affine=True)
    (2): Hardtanh (min_val=0, max_val=20, inplace)
    (3): Conv2d(32, 32, kernel_size=(21, 11), stride=(2, 1))
    (4): BatchNorm2d(32, eps=1e-05, momentum=0.1, affine=True)
    (5): Hardtanh (min_val=0, max_val=20, inplace)
  )
  (rnns): Sequential (
    (0): BatchRNN (
      (rnn): LSTM(672, 400, bias=False, bidirectional=True)
    )
    (1): BatchRNN (
      (batch_norm): SequenceWise (
      BatchNorm1d(400, eps=1e-05, momentum=0.1, affine=True))
      (rnn): LSTM(400, 400, bias=False, bidirectional=True)
    )
    (2): BatchRNN (
      (batch_norm): SequenceWise (
      BatchNorm1d(400, eps=1e-05, momentum=0.1, affine=True))
      (rnn): LSTM(400, 400, bias=False, bidirectional=True)
    )
    (3): BatchRNN (
      (batch_norm): SequenceWise (
      BatchNorm1d(400, eps=1e-05, momentum=0.1, affine=True))
      (rnn): LSTM(400, 400, bias=False, bidirectional=True)
    )
  )
  (fc): Sequential (
    (0): SequenceWise (
    Sequential (
      (0): BatchNorm1d(400, eps=1e-05, momentum=0.1, affine=True)
      (1): Linear (400 -> 29)
    ))
  )
)
Number of parameters: 11376368
Segmentation fault (core dumped)
```

**Comment by ryanleary** [comment]
That's not really enough information to diagnose what's going on. That said, given the fact that there's a seg fault, I'm inclined to think this is likely a pytorch issue.

**Questions:**
1. Is this on a GPU or CPU (the command line args you supplied indicate CPU, but I wanted to double check
2. OS Version
3. If GPU, what CUDA and cuDNN versions? what GPU?
4. Python version
5. Pytorch version (`import pytorch; print(pytorch.__version__)`)
6. SHA1 of deepspeech.pytorch you are using

Thanks

**Comment by mcernak** [comment]
Hi guys,
I had similar error, when running `train.py` on Debian GNU/Linux 8 (jessie), CPU, CUDA v 8.0, pytorch v. 0.1.12_2.
Segmentation fault appeared on `loss.backward()`.
Training with `--cuda` works well.

**Comment by ryanleary** [comment]
@supriyalk were you training on CPU or GPU?

**Comment by SeanNaren** [comment]
I run into a similar issue on my home machine, where after training i get a segmentation fault. I'll try to investigate

**Comment by ryanleary** [comment]
Closing due to no followup. Assumed fixed or worked around.

**Comment by pskrunner14** [comment]
https://github.com/SeanNaren/warp-ctc/issues/62
